app/controllers/verify/step_controller.rb

```diff
@@ -8,5 +8,28 @@ class StepController < ApplicationController
     before_action :confirm_idv_attempts_allowed
 
     helper_method :step
+
+    private
+
+    def step_name
+      raise NotImplementedError, 'must implement step_name method'
+    end
```

Comment:
IMO, each controller should have a corresponding route. If a controller is not directly called, then it should probably not be a controller (except for ApplicationController of course).